src/tests/e2e.test.tsx

```diff
@@ -413,5 +423,41 @@ describe('@shopify/preact-testing', () => {
       const wrapper = mountWithContext(<Message />);
       expect(spy.mock.calls).toEqual([['render'], ['render'], ['afterMount']]);
     });
+
+    it('waits on an async afterMount', async () => {
```

Comment:
Added this to make sure `async afterMount` functions work, I think it's a useful test despite them not being broken in the first place so I left it in.